walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/datasource/network/apis/HttpAgentOpenId4VciApi.kt

```diff
@@ -10,16 +10,32 @@ import kotlinx.serialization.json.Json
  * Api class to perform OpenId4VCI related network operations using the provided HttpAgent, utils
  * and json serializer to convert the network response to Issuance related model.
  */
-internal class HttpAgentOpenId4VciApi(private val agent: IHttpAgent,
-                                      private val httpAgentUtils: HttpAgentUtils,
-                                      private val json : Json
+internal class HttpAgentOpenId4VciApi(
+    private val agent: IHttpAgent,
+    private val httpAgentUtils: HttpAgentUtils,
+    private val json: Json
 ) {
 
     suspend fun getOpenID4VCIRequest(overrideUrl: String): Result<IResponse> {
-        return agent.get(overrideUrl, httpAgentUtils.combineMaps(
-            httpAgentUtils.defaultHeaders(),
-            mapOf(
-                Constants.PREFER to "oid4vci-interop-profile-version=0.0.1"
-            )))
+        return agent.get(
+            overrideUrl, httpAgentUtils.combineMaps(
```

Comment:
these two methods like almost the same, any chance we can move logic into a private fun to clean it up?